packages/web/server/queries/complex/assets/index.ts

```diff
@@ -139,6 +141,40 @@ function simplifyAssetListForDisplay(
   return assetListAssets.map(makeMinimalAsset);
 }
 
+export async function mapRawAssetsToCoinPretty({
+  rawAssets,
+  calculatePrice,
+}: {
+  rawAssets?: {
+    amount: string | number;
+    denom: string;
+  }[];
+  calculatePrice?: boolean;
+}): Promise<(CoinPretty & { fiatValue?: Dec })[]> {
```

Comment:
At complex layer, I'd prefer PricePretty over Dec for fiat value.

Also, long term I'm thinking we should create our own unit package, and rename *Pretty to just *.